PodcastGenerator/index.php

```diff
@@ -9,6 +9,17 @@
 ############################################################
 session_start();
 require 'core/include.php';
+
+// Backwards compatibility: Redirect pre-3.0 archive pages to
+// categories.php.
+if(isset($_GET['p'])) {
+    if($_GET['p'] == 'archive') {
+        $redirect_url = $config['url'] . 'categories.php?cat=' . $_GET['cat'];
+        header('Location: ' . $redirect_url, TRUE, 301);
```

Comment:
Please remove the last two parameters. I know they are good but currently we don't use them at any function.
This is change of another PR.

Reply:
Done.